exercises/practice/leap/.approaches/config.json

```diff
@@ -0,0 +1,32 @@
+{
+  "introduction": {
+    "authors": ["meatball133"],
+    "contributors": ["BNAndras"]
+  },
+  "approaches": [
+    {
+      "uuid": "5d42dc83-2473-425a-90bd-bf03f92b8c8b",
+      "slug": "divisible-by",
+      "title": "Use divisible-by",
+      "blurb": "Use divisible-by to check number.",
+      "authors": ["meatball133"],
+      "contributors": ["BNAndras"]
+    },
+    {
+      "uuid": "9952fef5-9f2f-4575-94fc-bc4e96593cd6",
+      "slug": "case",
+      "title": "Uses Case",
+      "blurb": "Use Case to check numbers.",
+      "authors": ["meatball133"],
+      "contributors": []
+    },
+    {
+      "uuid": "5a0d8b86-57d7-4418-a730-09be7c9fbaa7",
+      "slug": "proc",
+      "title": "Uses Proc",
+      "blurb": "Use Proc to check number.",
```

Comment:
I'm not entirely sure about these. Maybe @BNAndras has an idea for a more descriptive version?

Reply:
I’m stumped. Perhaps “Use the leap year code as data”? I’d want students to have a rough approximation of what a proc might be in case they don’t know the concept ahead of time.